ietf/secr/rolodex/forms.py

```diff
@@ -85,7 +85,7 @@ def clean_user(self):
         if user:
             # if Django User object exists return it, otherwise create one
             try:
-                user_obj = User.objects.get(username=user)
+                user_obj = User.objects.get(username__iexact=user)
             except User.DoesNotExist:
                 user_obj = User.objects.create_user(user,user)
```

Comment:
May want to force lower() as with views

Reply:
ok, again the arguments here should be explicit, not positional.